src/live_restore/live_restore_fs.c

```diff
@@ -1581,6 +1595,71 @@ __validate_live_restore_path(WT_FILE_SYSTEM *fs, WT_SESSION_IMPL *session, const
     return (fh->close(fh, (WT_SESSION *)session));
 }
 
+/*
+ * __wt_live_restore_setup_recovery --
+ *     Perform necessary setup steps prior to recovery running, this is largely copying log files
```

Comment:
Run on sentence. Change to "...prior to recovery running. This is largely..."

Reply:
042d20ceb